src/components/four-up-overlay.tsx

```diff
@@ -1,28 +1,67 @@
+import { inject, observer } from "mobx-react";
 import * as React from "react";
 import { BaseComponent, IBaseProps } from "./base";
+import { DocumentViewMode } from "./teacher/teacher-group-tab";
 
 import "./four-up-overlay.sass";
+import { DocumentModelType } from "../models/document/document";
```

Comment:
nit: we've generally maintained the convention that the ".sass" import is the last import, so this should be moved up with the "DocumentViewMode" import.